### PR TITLE
Added ondelete and onupdate to the generated reference arrays

### PR DESCRIPTION
**Pull request by rianorie**
also see phalcon/cphalcon#11162

### CODE REVIEW

File: scripts/Phalcon/Mvc/Model/Migration.php [hunk 1]
```diff
@@ -321,6 +321,8 @@ public static function generate($version, $table, $exportData = null)
                 $columns[] = "'" . $column . "'";
             }
 
+            var_dump($dbReference);
```
Comment (oscarmolinadev) [inline scripts/Phalcon/Mvc/Model/Migration.php]: Yep! var_dump here :)

**Comment by rianorie** [comment]
it was really late/early, sorry!

**Comment by sergeyklay** [comment]
`master` branch?

**Comment by rianorie** [comment]
first time in months I have time and it just ain't working out, man. ignore it! I'll rebase against 2.0.x after I finish beating my head against the wall -_-

**Comment by sergeyklay** [comment]
:neutral_face:

**Comment by rianorie** [comment]
don't take that too literal :p life isn't THAT bad heh, just.. first the damned var_dump then `master`.. I'm just waiting on the next one now ;)

**Pull request closed without merging**